src/main/scala/com/normation/rundeck/plugin/resources/rudder/DataModel.scala

```diff
@@ -41,6 +41,7 @@ sealed trait ApiVersion { def value: String }
  */
 case object ApiV6  extends ApiVersion { val value = "6"  }
 case object ApiV12 extends ApiVersion { val value = "12" }
+case object ApiV18 extends ApiVersion { val value = "18" }
```

Comment:
I thing here, we can even remove ApiV6, which is unbelieveblaly outdated, keep v12 in case something breaks in unforseen side effects, and replace `ApiV18` by `ApiLatest` with value `latest`. We have an excellent track record with backward compat of API, it should be ok here, too.